src/managers/PresenceManager.js

```diff
@@ -40,7 +40,7 @@ class PresenceManager extends BaseManager {
     const presenceResolvable = super.resolve(presence);
     if (presenceResolvable) return presenceResolvable;
     const UserResolvable = this.client.users.resolveID(presence);
-    return super.resolve(UserResolvable) || null;
+    return super.resolve(UserResolvable) ?? null;
```

Comment:
`super.resolve` should already return null if this would evaluate to null if I'm not mistaken, why not  just remove it completely.